herd/libdir/asl-pseudocode/implementations.asl

```diff
@@ -609,3 +609,55 @@ getter SCTLR_EL1[] => SCTLRType
 begin
   return Zeros(64);
 end
+
+// InstructionSynchronizationBarrier()
+// ===================================
+func InstructionSynchronizationBarrier()
+begin
+  primitive_isb();
+  return;
```

Comment:
FYI, `return;` is not mandatory when not returning anything in ASLv1

Reply:
Thanks.